web/static/js/views/cards/show.js

```diff
@@ -3,13 +3,21 @@ import { connect }        from 'react-redux';
 import CardModal          from '../../components/cards/modal';
 import Actions            from '../../actions/current_card';
 
+
 class CardsShowView extends React.Component {
   componentDidMount() {
     const { dispatch, params } = this.props;
 
-    console.log(params);
 
-    dispatch(Actions.showCard(params.id[0], params.id[1]));
+    dispatch(Actions.showCard(this.getCard(params.id[1])));
+  }
+
+  getCard(id) {
+    let cId = +id;
+    let cards = [];
+    this.props.currentBoard.lists.forEach((list) => { cards = cards.concat(list.cards) });
+
+    return cards.find( (c) => { return c.id === cId  });
```

Comment:
Here we could check agains `+id` instead of creating the new var in line 16, what do you think?
```
return cards.find( (c) => { return c.id === +id  });
```